solutions/client-setup/namespaces/client-name-networking.yaml

```diff
@@ -76,6 +76,23 @@ spec:
   role: roles/dns.admin
   member: "serviceAccount:client-name-networking-sa@client-management-project-id.iam.gserviceaccount.com" # kpt-set: serviceAccount:${client-name}-networking-sa@${client-management-project-id}.iam.gserviceaccount.com
 ---
+# Grant GCP role DNS Records Admin to GCP SA
```

Comment:
please follow this convention for the comment https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/blob/71725d1563e6743a9b6ebdc5c9b99b0fb06018ea/solutions/client-project-setup/namespaces/project-id-tier3.yaml#LL83C1-L83C73

Reply:
Grant GCP role Tier2 DNS Record Admin to GCP SA on Core DNS Project

Reply:
Edited the wrong resource, but I think it's ok now.